app/containers/EmojiPicker/EmojiCategory.js

```diff
@@ -42,10 +44,56 @@ export default class EmojiCategory extends React.Component {
 
 		this.size = Math.min(width || widthWidth, windowHeight) / (emojisPerRow || EMOJIS_PER_ROW);
 		this.emojis = props.emojis;
+		this.state = {
+			isModalVisible: false,
+			modalX: 0,
+			modalY: 0,
+			modalData: []
+		};
 	}
 
 	shouldComponentUpdate() {
-		return false;
+		return true;
+	}
+
+	componentWillUnmount() {
+		this.setState({ isModalVisible: false });
+	}
+
+	getModelStyle() {
+		const { modalX, modalY } = this.state;
+		const { width } = Dimensions.get('window');
+		const widthOfModal = this.size * 5.2;
```

Comment:
Where 5.2 came from?

Reply:
trial and error

Reply:
you can see it as 5 elements are displaying and .2 as padding